src/assets/styles/style.scss

```diff
@@ -0,0 +1,1081 @@
+      font-size: 18px;
+      margin-bottom: 5px;
+      line-height: 20px;
+    }
+    @media (max-width: 767px) {
+      font-size: 14px;
+      margin-bottom: 5px;
+      line-height: 20px;
+      text-align: center;
+    }
+  }
+
+  &__list {
+    margin-left: 20px;
+    font: {
+      weight: bold;
+      size: 14px;
+    }
+    @media (max-width: 1199px) {
+      display: flex;
+      flex-direction: row;
+      justify-content: space-between;
+    }
+    @media (max-width: 767px) {
+      display: unset;
+      margin-left: unset;
+    }
+  }
+
+  &__item {
+    margin-bottom: 26px;
+
+    &:not(:first-child):hover {
+      background-color: $bgColor;
+      line-height: 21px;
+      margin-bottom: 19px;
+      margin-left: -20px;
+      font: {
+        weight: bold;
+        size: 14px;
+      }
+      .categories__link {
+        color: $btnTextColor;
+        padding-left: 20px;
+      }
+    }
+
+    &_active {
+      background-color: $bgColor;
+      text-align: center;
+      width: 68px;
+      line-height: 21px;
+      margin-bottom: 28px;
+      margin-left: -20px;
+      font: {
+        weight: bold;
+        size: 14px;
+      }
+    }
+
+    @media (max-width: 1199px) {
+      &:not(:first-child):hover {
+        margin-bottom: 27px;
+        margin-left: 0px;
+        .categories__link {
+          padding-left: 0px;
+        }
+      }
+    }
+
+    @media (max-width: 767px) {
+      font-size: 12px;
+      line-height: 14px;
+      margin-bottom: 10px;
+      &:not(:first-child):hover {
+        margin-left: -20px;
+        line-height: 14px;
+        margin-bottom: 10px;
+        font: {
+          weight: bold;
+          size: 12px;
+        }
+        .categories__link {
+          color: $btnTextColor;
+          padding-left: 20px;
+        }
+      }
+    }
+  }
+
+  &__link {
+    color: $regularTextColor;
+    text-decoration: none;
+    &_active {
+      color: $btnTextColor;
+    }
+  }
+}
+
+.gallery {
+  display: grid;
+  grid-template-columns: repeat(3, 260px);
+  grid-column-gap: 35px;
+  margin-top: 12px;
+
+  @media (max-width: 1199px) {
+    grid-template-columns: repeat(3, minmax(200px, 260px));
+    grid-column-gap: 10px;
+    align-self: center;
+  }
+
+  @media (max-width: 767px) {
+    grid-template-columns: repeat(2, minmax(200px, 260px));
+    grid-column-gap: 5px;
+  }
+
+  @media (max-width: 500px) {
+    grid-template-columns: repeat(1, minmax(200px, 260px));
+    grid-column-gap: 5px;
+  }
+
+  &__column {
+    &_hide2 {
+      @media (max-width: 500px) {
+        display: none;
+      }
+    }
+    &_hide3 {
+      @media (max-width: 767px) {
+        display: none;
+      }
+    }
+  }
+
+  &__img {
+    margin-bottom: 25px;
+    @media (max-width: 1199px) {
+      width: 100%;
+      margin-bottom: 10px;
+    }
+  }
+}
+
+.form {
+  background-color: $btnTextColor;
+  @media (max-width: 767px) {
+    margin-left: 10px;
+    margin-right: 10px;
+  }
+
+  &__title {
+    margin-top: 11px;
```

Comment:
Will be better if you will use padding for the wrapper tag and use margin for the bottom indent.

Reply:
Done everywhere except one case in subscribe form.